src/Components/Video Card/video-card.jsx

```diff
@@ -1,32 +1,62 @@
-import { Link } from "react-router-dom";
-
+import { useNavigate } from "react-router-dom";
+import { useState, useEffect } from "react";
+import {useAuth} from '../../Context/authContext'
+import { addToWatchLaterData, deleteFromWatchLaterData } from '../../Utilities/JS/dataHandlers/watchLaterDataHandlers'
+import {useUserData} from '../../Context/userDataContext'
 import './video-card.css'
 
-const VideoCard = ({ video, cardDirection }) => {
-    
+const VideoCard = ({ video }) => {
+
+    const {title, thumbnailUrl, channelImg, views, videoLength, channelName, description, youtubeId, _id } = video
+  const navigate = useNavigate();
+
+    const {auth:{token ,isAuthorized}} = useAuth()
+    const {userData:{watchLaterData}, userDataDispatch} = useUserData()
+
+    const [showActionCard, setShowActionCard] = useState(false)
+    const [isInWatchLater, setIsInWatchLater] = useState(false)
+
+    useEffect(() =>{
+        watchLaterData.find((video) => video._id === _id) && setIsInWatchLater(true)
+    },[watchLaterData, _id]) 
+
     return (
         		
-
         <div className="card vertical">
-        <Link to='/video'>
-        <div className="card__product--img">
-            
-                <img className="img " src={video.thumbnailUrl} alt="video thumbnail"/>
-           
-            <span className="badge--text badge--grey">{video.videoLength}</span>
-        </div>
-        <div className="card__product--content">
-            <h2 className="card__product--title">{video.title}</h2>
-            <span className="card__product--category">{video.views} views</span>
-            <img className="avatar__round avatar--sm" src={video.channelImg} alt='channel logo'/>
-            <h4 className="card__product--company">{video.channelName} </h4>
-            <p className="card__product--description"> {video.description} </p>
+            <div onClick={() => navigate(`/video/${youtubeId}`)}>
+                <div className="card__product--img">
+                    <img className="img" src={thumbnailUrl} alt="video thumbnail"/>
+                    <span className="badge--text badge--grey">{videoLength}</span>
+                </div>
+                <div className="card__product--content">
+                    <h2 className="card__product--title">{title}</h2>
+                    <span className="card__product--category">{views} views</span>
+                    <img className="avatar__round avatar--sm" src={channelImg} alt='channel logo'/>
```

Comment:
You can use the channel name in the `alt` attribute.

Reply:
Thanks for the suggestion!